ansible/controller.yml

```diff
@@ -1,6 +1,12 @@
 ---
-# This playbook deploys Openwhisk Controllers.  
+# This playbook deploys Openwhisk Controllers.
 
 - hosts: controllers
+  vars:
+    host_group: "{{ groups['controllers'] }}"
+    name_prefix: "controller"
+    controller_index_base: 0
+    local_bookkeeping: controller.localBookkeeping
+    akka_enabled: "True"
```

Comment:
Can we abstract this to `join_cluster: True`? akka_enabled is only partially true, since the component will ofcourse still use akka, but it doesn't join the cluster.

Reply:
Can do.  Thanks for the clarification.